mipdb/database.py

```diff
@@ -503,6 +514,16 @@ def table_exists(self, table):
     def create_table(self, table):
         table.create(bind=self._executor)
 
+    @handle_errors
+    def grant_select_to_executor(self, table):
```

Comment:
Rename to `grant_select_access_rights` ?

Also I think it would be better to pass the user as a parameter. Move the logic of the executor outside this method.